src/test/java/org/sefglobal/scholarx/service/ProgramServiceTest.java

```diff
@@ -394,6 +401,12 @@ void updateMenteeData_withUnsuitableData_thenThrowBadRequest() {
         mentee.setState(EnrolmentState.APPROVED);
         mentor.setState(EnrolmentState.APPROVED);
         mentee.setAppliedMentor(mentor);
+        
+        Program program = new Program();
+        program.setId(programId);
+        program.setState(ProgramState.MENTEE_SELECTION);
```

Comment:
```suggestion
        program.setState(ProgramState.MENTEE_APPLICATION);
```
This will fix the test

Reply:
I changed it. Thank you.